src/client.rs

```diff
@@ -49,6 +56,18 @@ impl Crux {
             headers: self.headers.clone(),
         }
     }
+
+    /// A mock of `DockerClient` using `mockito = "0.26"`.
+    #[cfg(feature = "mock")]
+    pub fn docker_mock(&mut self) -> DockerClient {
```

Comment:
Is this necessary? I think the only change the user needs to do is to use the feature and add `mockito` to its code, it shouldn't need to change the client usage.

Reply:
Besides, it is the same code as the `docker_client` method

Reply:
I prefer it to be explicit and only available for the feature

Reply:
Okay then